src/types/index.ts

```diff
@@ -45,12 +45,14 @@ export type BetHistory = {
   index: number;
   market_id: string;
   proposition_id: string;
+  winningPropositionId: string | undefined;
```

Comment:
Why can winningPropositionId be undefined?

Reply:
It doesn't exist on races that haven't completed

Reply:
good question!